rts/Lua/LuaSyncedRead.cpp

```diff
@@ -7004,7 +7014,10 @@ int LuaSyncedRead::GetFeatureNoSelect(lua_State* L)
 /***
  *
  * @function Spring.GetFeatureResurrect
+ * Returns nil if no feature found with ID.
  * @param featureID integer
+ * @return string|""|nil featureDefName
+ * @return integer buildFacing facing of footprint, 0 - 3
```

Comment:
I've actually created a type alias for facing, looks like `FacingInteger` should be used instead of `integer`

https://beyond-all-reason.github.io/spring/lua-api/types/FacingInteger